cinema/serializers.py

```diff
@@ -1,24 +1,35 @@
 from rest_framework import serializers
+from rest_framework.validators import UniqueValidator
 
-from cinema.models import Movie
+from cinema.models import Actor, Genre, CinemaHall, Movie
 
 
-class MovieSerializer(serializers.Serializer):
-    id = serializers.IntegerField(read_only=True)
-    title = serializers.CharField(max_length=255)
-    description = serializers.CharField()
-    duration = serializers.IntegerField()
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name")
 
-    def create(self, validated_data):
-        return Movie.objects.create(**validated_data)
 
-    def update(self, instance, validated_data):
-        instance.title = validated_data.get("title", instance.title)
-        instance.description = validated_data.get(
-            "description", instance.description
-        )
-        instance.duration = validated_data.get("duration", instance.duration)
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ("id", "name",)
 
-        instance.save()
 
-        return instance
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ("id", "name", "rows", "seats_in_row")
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    actors = serializers.PrimaryKeyRelatedField(
+        many=True, queryset=Actor.objects.all()
+    )
+    genres = serializers.PrimaryKeyRelatedField(
+        many=True, queryset=Genre.objects.all()
+    )
+
+    class Meta:
+        model = Movie
```

Comment:
We'll learn these serializers in API Serilizer topic, for this you should still use Serializer because
<img width="730" alt="image" src="https://github.com/user-attachments/assets/929a2c65-0e65-40f9-a0ef-958a970fe962">